### PR TITLE
Build /managed page

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Moved the existing `/managed` page to `managed/apps`
- Created `/managed` page
- Created [Managed Services Contact us](https://docs.google.com/document/d/18bgYyreMlSzsMKI5AaTQU1Qjz8eEkVz41Du9q60ZRiY/edit#heading=h.dxn10eelvkoc) modal

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/managed
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please check page against [copy doc.](https://docs.google.com/document/d/19GoMudyrd9Zt1e7kfj-koL-Lt0RGYzWHRPuuKYUPvqw/edit#heading=h.mnstvknzgivc)
- Check against [design](https://github.com/canonical-web-and-design/web-squad/issues/4319)

## Issue / Card

Fixes [#4320](https://github.com/canonical-web-and-design/web-squad/issues/4320)

## Screenshots
Managed IT Services for Openstack, Kubernetes and open source applications | Ubuntu (2021-08-20 09-18-45)

### CODE REVIEW

**Comment by minkyngkm** [comment]
The cloud image with applications will be updated.

**Comment by anasereijo** [comment]
hey @minkyngkm was this based on a design or just the copydoc? If so, can you please leave a link to the design issue or zeplin link? thank you

**Comment by minkyngkm** [comment]
> hey @minkyngkm was this based on a design or just the copydoc? If so, can you please leave a link to the design issue or zeplin link? thank you

Hi @anasereijo! The design issue is [here](https://github.com/canonical-web-and-design/web-squad/issues/4319).

**Comment by anasereijo** [comment]
> > hey @minkyngkm was this based on a design or just the copydoc? If so, can you please leave a link to the design issue or zeplin link? thank you
> 
> Hi @anasereijo! The design issue is [here](https://github.com/canonical-web-and-design/web-squad/issues/4319).

Thanks! I will add it to the QA steps.

**Comment by minkyngkm** [comment]
> > > hey @minkyngkm was this based on a design or just the copydoc? If so, can you please leave a link to the design issue or zeplin link? thank you
> > 
> > 
> > Hi @anasereijo! The design issue is [here](https://github.com/canonical-web-and-design/web-squad/issues/4319).
> 
> Thanks! I will add it to the QA steps.

Oh I missed it in the QA steps! Thank you 👍

**Comment by anasereijo** [comment]
hey @minkyngkm this is looking good. Just some (hopefully) small things:

- [ ] I can see you copied what's on the copydoc but please make the whole text part of the link:
Screenshot 2021-08-19 at 15 10 10
Screenshot 2021-08-19 at 15 10 19


(please make sure you pick an .svg version of logos when possible - there are a few pngs and jpgs available but we should always use svgs)
- [ ] replace okcupid logo with [this one](https://assets.ubuntu.com/v1/c6a8ecb7-okcupid_logo.svg)
- [ ] liberty logo replace with [this](https://assets.ubuntu.com/v1/2f1934f0-Liberty-Global-Logo.svg)

- [ ] A frictionless cloud journey
Screenshot 2021-08-19 at 15 03 39

section with the 3 illustrations
- [ ] use[ this illustration](https://assets.ubuntu.com/v1/317680dc-CLOUD+APPS.svg) for the last one
- [ ] if you see the design we use links and not buttons

- [ ] the chevron should always be part of the link
Screenshot 2021-08-19 at 15 06 03

- [ ] hide the following images on small screens:
Screenshot 2021-08-19 at 15 07 40
Screenshot 2021-08-19 at 15 07 55
Screenshot 2021-08-19 at 15 08 06

**Comment by minkyngkm** [comment]
Hey @anasereijo Thanks for reviewing! :) 
I have updated it all as you have suggested. Hopefully it looks good better! 
One thing to ask though. I have tried to update all the logos to .svg but there were only .png available for some logos. Would that be okay? Or Would there any way that I can find the .svg files other than managers.assets.ubuntu.com?

**Comment by anasereijo** [comment]
This is looking good! thanks for making the changes

**Comment by anasereijo** [comment]
> Hey @anasereijo Thanks for reviewing! :)
> One thing to ask though. I have tried to update all the logos to .svg but there were only .png available for some logos. Would that be okay? Or Would there any way that I can find the .svg files other than managers.assets.ubuntu.com?

in that case, we have to try finding the .svg version on google and then upload it on the assets server. You can ask the designer's help with that, so don't spend too much time trying to find them. In some cases we can't seem to find a svg so we do our best to pick a better resolution png :)

File: templates/shared/forms/interactive/managed-services.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+            hi_def=True,
+            loading="auto",
+            ) | safe
+          }}
+            </div>
+        </div>
+        <a class="js-close p-button--neutral" href="">Close</a>
+    </div>
+</div>
+</div>
```
Comment (carkod) [inline templates/shared/forms/interactive/managed-services.html:155]: This indentation is not correct? Which `div` is it closing?

**Review by carkod — APPROVED** [review]
LGTM, just one comment.

File: templates/shared/forms/interactive/managed-services.html (hunk 1 above)
Comment (minkyngkm) [reply templates/shared/forms/interactive/managed-services.html]: > This indentation is not correct? Which `div` is it closing?

It should have been removed... Fixed!

**Pull request merged**